tests/ftests/073-pybindings-cgroup_add_all_controllers-v2.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# cgroup_add_all_controllers() test using the python bindings
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Tom Hromatka <tom.hromatka@oracle.com>
+#
+
+from libcgroup import Cgroup, Version, Mode
+from cgroup import Cgroup as CgroupCli
+import consts
+import ftests
+import sys
+import os
+
+
+PARENTCG = '073cgggetcg'
+CGNAME = '{}/child'.format(PARENTCG)
+CONTROLLERS = ['cpu', 'memory', 'pids']
+SUBTREE_CONTROL = 'cpu'
+
+def prereqs(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if config.args.container:
+        result = consts.TEST_SKIPPED
+        cause = 'This test cannot be run within a container'
```

Comment:
This should probably work in a container, but I didn't try very hard to get it working.  Thoughts, @kamalesh-babulal ?

Reply:
It should work in the container too, let me try it out in the container too ensure there are no issues.